protected/models/Patient.php

```diff
@@ -57,8 +57,10 @@
  * @property SecondaryDiagnosis[] $secondarydiagnoses
  * @property EthnicGroup $ethnic_group
  * @property CommissioningBody[] $commissioningbodies
- * @property SocialHistory $socialhistory
- * @property TrialPatient[] trials
+ * @property \OEModule\OphCiExamination\models\SocialHistory $socialhistory
+ * @property TrialPatient[] $trials
```

Comment:
Is this the right type? Or maybe the variable has been named incorrectly.

Reply:
Everything in OphCiExamination is namespaced, so I believe it has to be declared in this way.